assets/css/styles.css

```diff
@@ -721,14 +738,291 @@ HAMBURGER MENU DISPLACED DUE TO LINTERS ERROR
+  --  Add .popup button and .popup h1, h2, h3 and p to respective
+      class declarations in the different sections
+  - Add the class open-popup (to be added in the javascript open function)
+*/
+.modal-container {
+  width: 100vw;
+  height: 100vh;
+  position: relative;
+  top: 0;
+}
+
+.popup-modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  background: var(--my-competences-boxes-color);
+  z-index: 1;
+  filter: opacity(95%);
+  overflow-y: hidden;
+  overscroll-behavior: none;
+}
+
+.project-popup {
+  width: 100%;
+  position: absolute;
+  background: var(--image-placeholder-color1);
+  border-radius: 1%;
+  top: 0;
+  left: 0;
+  text-align: center;
+  padding: 0;
+  transition: transform 0.4s, visibility 0.4s, top 0.4s;
+  visibility: visible;
+  z-index: 100;
+  backdrop-filter: blur(15px);
+  backdrop-filter: brightness(60%);
+  backdrop-filter: contrast(40%);
+}
+
+.js-project-popup-img {
+  width: 100%;
+  border-bottom: 0;
+}
+
+.project-popup-desktop-img {
+  width: 100%;
+  height: 50%;
+  margin-top: 3%;
+  border-radius: 1%;
+  border: 1px solid white;
+}
+
+.project-popup-mobile-img {
+  width: 100%;
+  height: 40% !important;
+  margin-top: 3%;
+  border-radius: 1%;
+  border: 1px solid white;
+}
+
+/*
+.project-popup-desktop-img img,
+.project-popup-mobile-img img {
+  width: 100% !important;
+  height: 100% !important;
+}
+*/
+
+.open-project-popup {
+  width: 100%;
+  height: 97vh;
+  top: 3vh;
+  left: 0%;
+  transform: translate(-250%, -250%) scale(0.1);
+  position: fixed;
+  visibility: hidden;
+  z-index: 1000;
+  background-color: var(--image-placeholder-color);
+}
+
+.open-project-popup .active {
+  visibility: visible;
+}
+
+.project-card-js {
+  position: fixed;
+  width: 80%;
+  height: 80vh;
+  top: 10%;
+  left: 10%;
+  background-color: white;
+  z-index: 900;
+}
+
+/*
+  STYLING THE JS CONTAINER
+*/
+
+.js-button-close {
+  position: absolute;
+  width: 5%;
+  height: 5%;
+  top: 0%;
+  right: 5%;
+  margin: 0;
+  border: 1px solid var(--image-placeholder-color);
+  border-radius: 5px;
+  background: url(../../images/popup/close.svg) no-repeat center;
+  align-self: center;
+}
+
+.js-text-container {
+  width: 90%;
+  height: 100%;
+  padding: 0.1% 5%;
+  margin-top: 5%;
+}
+
+.js-title-div {
+  width: 100%;
+  height: 10%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.js-mobile-link-list {
+  width: 100%;
+  height: 10%;
+  display: flex;
+  justify-content: center;
+  flex-wrap: nowrap;
+}
+
+.js-liveLink-img {
+  width: 20%;
+  height: 20%;
+  background: url(../../images/popup/icon-export.svg) no-repeat center;
+}
+
+.js-sourceLink-img {
+  width: 100%;
+  height: 100%;
+  background: url(../../images/popup/icon-gitHub.svg) no-repeat center;
+}
+
+.js-liveLink-button {
+  /*
+  width: 20%;
+  height: 20%;
+  */
+  background: url(../../images/popup/button-see-live.svg) no-repeat center;
+  background-size: 100%;
```

Comment:
- [ ] As requested by the previous reviewer please make your design similar to the Figma design as buttons and descriptions are not showing on the pop-up window. See the image below for reference
- [ ] When I scroll then blur part is scrolling and the buttons and description is not showing please work on it and make sure to match the design.
### Figma design
![fimga design](https://user-images.githubusercontent.com/81727988/233696031-e6bfe46c-dd80-4849-84c1-c8157cc34981.png)

### Your design
![his](https://user-images.githubusercontent.com/81727988/233696052-4e029d24-737e-43e6-8cf9-45b5de4068fc.png)

Reply:
Done @roshan-bajgain 
Image height fixed